### PR TITLE
13.0 mig account analytic required

### PR DESCRIPTION
**Pull request by jimhoefnagels**

### CODE REVIEW

**Review by schout-it — APPROVED** [review]
lgtm

**Review by CasVissers-360ERP — APPROVED** [review]
LGTM!

**Comment by jimhoefnagels** [comment]
Could this PR be merged please?
thanks

File: oca_dependencies.txt
Comment (ivantodorovich) [inline oca_dependencies.txt]:
```suggestion
account-financial-tools
```

If this PR depends on another, we should wait for the other to be merged first.
In this case I think it's already merged.

You can remove the comments on this file aswell

**Review by ivantodorovich — CHANGES_REQUESTED** [review]

**Review by ivantodorovich — APPROVED** [review]
LGTM!

I would ask you to: 

> Squash administrative commits (if any) with the previous commit for reducing commit noise. They are named as "[UPD] README.rst", "[UPD] Update $MODULE.pot", "Update translation files" and similar names, and comes from OCA-git-bot, oca-travis or oca-transbot.

https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0

Otherwise :+1:

**Comment by jimhoefnagels** [comment]
Hi @pedrobaeza 
Can this PR be merged? Thanks

**Comment by pedrobaeza** [comment]
Sorry, I'm no more OCA's community manager for handling these things. Any way, let's do it for one more time:

/ocabot merge

**Comment by sbidoul** [comment]
> Sorry, I'm no more OCA's community manager for handling these things. Any way, let's do it for one more time

Note that strictly speaking, merging a PR is the responsibility of the addon maintainer or [PSC members](https://odoo-community.org/psc-teams/accounting-9). So this case the right group to ping is @OCA/accounting-maintainers.

**Pull request merged**
